packages/admin/admin/src/common/MainContent.tsx

```diff
@@ -37,16 +41,28 @@ const Root = createComponentSlot("main")<MainContentClassKey, OwnerState>({
         ${ownerState.disablePaddingTop &&
         css`
             padding-top: 0;
+
+            ${theme.breakpoints.up("md")} {
+                padding: 0;
+            }
```

Comment:
Is this correct? Now disablePadding, disablePaddingTop and disablePaddingBottom all disable all paddings md and up.

Reply:
No 😅